packages/components/src/lib/controllers/app-controller.js

```diff
@@ -878,16 +878,28 @@ export default class AppController {
   /**
    * Gets language name details by either language ID (a symbol) or language code (string)
    *
-   * @param {symbol|string} language - Either language ID or language code (see constants in `data-models` for definitions)
+   * @param {Language | symbol|string} language - A language for which a name should be retrieved.
```

Comment:
should we note that symbol and string are deprecated value types?

Reply:
That's a good idea, I will add that.